src/hotspot/cpu/aarch64/macroAssembler_aarch64.hpp

```diff
@@ -1296,11 +1296,41 @@ class MacroAssembler: public Assembler {
   void kernel_crc32c_using_crc32c(Register crc, Register buf,
         Register len, Register tmp0, Register tmp1, Register tmp2,
         Register tmp3);
+
+  void ghash_modmul_wide (int index, FloatRegister result,
```

Comment:
Is there definition and reference of this?

Reply:
Removed.